### ISSUE TITLE
support for milliseconds

### ISSUE THREAD

**Issue by thijstriemstra** [ungoldman/format-duration#4]
The documentation states

```
// anything under a second is rounded down to zero
format(999) // '0:00'
```

but I need to format a duration that includes ms, e.g. 01:23:235

Is this possible?

**Comment by ungoldman** [comment]
source code is easily modifiable if you need something different. if you want to extend the library to optionally support milliseconds a PR is also welcome. good luck ✌️

**Comment by ungoldman** [comment]
Added in https://github.com/ungoldman/format-duration/pull/16, shipped in [3.0.0](https://github.com/ungoldman/format-duration/releases/tag/v3.0.0)
